src/main/java/se/bjurr/violations/lib/example/MyClass.java

```diff
@@ -9,6 +9,8 @@ void npe(String a, String b) {
   } else {
```

Comment:
**Rule**: EQ_ALWAYS_TRUE

equals method always returns true&#10;&#10;&#10;  &lt;p&gt; This class defines an equals method that always returns true. This is imaginative, but not very smart.&#10;Plus, it means that the equals method is not symmetric.&#10;&lt;/p&gt;&#10;

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1079697360>*